packages/custom-webpack/src/custom-webpack-builder.spec.ts

```diff
@@ -109,6 +109,33 @@ describe('CustomWebpackBuilder', () => {
     }
   });
 
+  it('should load the TS file specified in configuration', async () => {
```

Comment:
This test makes sure the TS file is loaded but it doesn't really check whether it is transpiled or not.  
Maybe it's worth to make slightly different test for this use case and import a real ts file.

Reply:
Yes, I agree with you. I will try to find a workaround.

Reply:
Moved webpack TS config to the e2e app.